### ISSUE TITLE
Order of polynomial coefficients

### ISSUE THREAD

**Issue by KronosTheLate** [JuliaDSP/DSP.jl#415]
I am generally confused around weather one should provide polynomial coeficcients for the lowest order term, or the higest order term, first. It varies from API to API. But in the documentation for DSP.jl, it seems like the same function does *_both_*! See picture below:
![image](https://user-images.githubusercontent.com/61620837/112460487-3e34f680-8d5f-11eb-87eb-e9474e2f0ed2.png)

Is this not an error? How would the function know which of the two seemingly contradictory definitions to use, based on `a` and `b`? Why not have the order of terms be the same?

**Comment by martinholters** [comment]
Yes, this used to be a source of confusion. On master (since #284), we now have `PolynomialRatio{:z}` and `PolynomialRatio{:s}` to distinguish the two cases. The documentation still needs updating, though, so this issue does have its justification.

**Comment by KronosTheLate** [comment]
I have since realized that one simply divides through by s^(m or n  - 1) and replaces s with z to get to the lower line, so that it makes mathematical sense. I guess I was just not fluent enough en going back and forth between representation of s to positive powers and z to negative powers.

So the docs are not wrong, but if I was confused I will probably not be the last.

But, huge shoutout to the quality of this package. I have been using it in my DSP course, which bases itself on MatLab, and I am keeping up no problem. I even think that DSP.jl is better in several cases - really amazing work, much appericated.

**Comment by KronosTheLate** [comment]
Friendly bump to update the docs.
